src/main/webapp/app/resources/styles/sass/management/_discoveryView.scss

```diff
@@ -1,4 +1,8 @@
 
 .add-field-group .btn {
   margin-top: 23px;
+}
+
+.clickers {
+  margin-top: 23px;
 }
```

Comment:
could use the terminating newline at the end of the file.
(yea, this lack of one is pre-existing, but might as well address it).